sections/exchange/hooks/useExchange.tsx

```diff
@@ -486,8 +646,14 @@ const useExchange = ({
 
 				const gasPriceWei = gasPriceInWei(gasPrice);
 
-				if (txProvider === '1inch') {
-					tx = await swap1Inch(quoteCurrencyKey!, baseCurrencyKey!, quoteCurrencyAmount, slippage);
+				if (txProvider === '1inch' && tokensMap != null) {
+					tx = await swap1Inch(
+						tokensMap[quoteCurrencyKey!].address,
```

Comment:
@evgenyboxer i think if we haven't we should include this

```
referrerAddress
OPTIONAL
string
referrer's address
``` 

and go onto the volume partners

Reply:
Will add in another PR after